docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -121,12 +121,22 @@ Going beyond that, you can specify an individual test method like this::
 Running the Selenium tests
 --------------------------
 
-Some tests require Selenium and a Web browser (Firefox, Google Chrome, or
+Some tests require Selenium(>=2.45) and a Web browser (Firefox, Google Chrome, or
```

Comment:
I'm not concerned about mentioning a minimum version of selenium because as fast as browsers are updated, older versions of selenium stop working fairly quickly in my experience. We can assume testers use the latest version.